oe_content.module

```diff
@@ -13,12 +13,21 @@ use Drupal\Core\Field\FieldStorageDefinitionInterface;
 use Drupal\link\LinkItemInterface;
 
 /**
- * Implements hook_entity_base_field_info().
+ * Implements hook_entity_bundle_field_info().
  */
-function oe_content_entity_base_field_info(EntityTypeInterface $entity_type) {
+function oe_content_entity_bundle_field_info(EntityTypeInterface $entity_type, $bundle, array $base_field_definitions) {
```

Comment:
This hook cannot be implemented in the same way as the hook_entity_base_field_info() one.